cmake/Dependencies.cmake

```diff
@@ -91,7 +91,7 @@ if(BUILD_BENCHMARK)
 endif()
 
 # Find or download/install rocm-cmake project
-find_package(ROCM QUIET CONFIG PATHS /opt/rocm)
+find_package(ROCM QUIET CONFIG PATHS ${CMAKE_INSTALL_PREFIX})
```

Comment:
Change to avoid using CMAKE_INSTALL_PREFIX.

Reply:
find_package() by searches the CMAKE_INSTALL_PREFIX path as part of the search list.
So then I will remove this change and keep /opt/rocm

Reply:
This is in general a design choice for all the libraries, how to specify which ROCM installation to use in a system. My main point is that a user may want to control CMAKE_INSTALL_PREFIX to point to their home directory as an example. ROCM_PATH works for me but more people should vet this decision.

Reply:
Yes, I agree to your point to keep the CMAKE_INSTALL_PREFIX and rocm module search path different. I will update the change and add ROCM_PATH along with /opt/rocm in the PATHS.